pypgstac/setup.py

```diff
@@ -0,0 +1,55 @@
+"""pypgstac: python utilities for working with pgstac."""
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+install_requires = [
+    "smart-open==4.2.*",
+    "orjson>=3.5.2",
+    "python-dateutil==2.8.*",
+    "fire==0.4.*",
+    "psycopg[binary]==3.0.*",
+    "psycopg-pool==3.1.*",
```

Comment:
I think how we handle these deps with the plan of making pypgstac a dependency of stac-fastapi.pgstac is still outstanding.